packages/teams-js/src/internal/communication.ts

```diff
@@ -320,6 +324,34 @@ export function sendMessageToParentAsync<T>(actionName: string, args: any[] | un
   });
 }
 
+/**
+ * @hidden
+ * Send a message to parent requesting a MessageChannel Port.
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export function requestPortFromParent(actionName: string, args: any[] | undefined = undefined): Promise<MessagePort> {
```

Comment:
```suggestion
export function requestPortFromParent(apiVersionTag: string, actionName: string, args: any[] | undefined = undefined): Promise<MessagePort> {
```

We have another engineer on our team currently working on converting all of our communication functionality to take in apiVersionTag here as well, so that it can be passed down from the caller. See `sendMessageToParentWithVersion` for an example of how that works. Function should do something similar to that (verify tag format, etc.) 

The tag that `messageChannels` passes in should be a combination of ApiVersionNumber.V_2 (because this function was first called in v2 of TeamsJS) and the actionName.

You can look at callers of `sendMessageToParentWithVersion` to see how to build tags correctly.

If we introduce new functionality or alternate signatures of this in the future we can update the tags to keep them straight in our telemetry.